index.js

```diff
@@ -0,0 +1,80 @@
+const hamburguerMenuIcon = document.querySelector('.three-bar-menu');
+// const crossIcon = document.querySelector('.crossIcon');
+
+hamburguerMenuIcon.addEventListener('click', () => {
+  // console.log('test');
+
+  const header = document.querySelector('header');
+  const body = document.querySelector('body');
+  const popUpMenuDiv = document.createElement('div');
+  const unorderedList = document.createElement('ul');
+  const crossIcon = document.createElement('a');
+  // const bottomLine = document.createElement('span');
+
+  const headerBar = document.querySelector('.header-bar');
+  const mainSection = document.querySelector('.main-section');
+  // let mainContainerProjects = document.querySelector(
+  //   '.main-container-projects'
+  // );
+
+  // add a class to the anchor tag
```

Comment:
- [ ] [OPTIONAL]
There are commented code blocks found in your code on the repository. Kindly endeavor to remove **all** commented code from your code before pushing it to the repository so as to keep your code well organized.